src/lib.rs

```diff
@@ -1573,8 +1588,7 @@ impl Iterator for ChunkRawIter<'_> {
     type Item = (*mut u8, usize);
     fn next(&mut self) -> Option<(*mut u8, usize)> {
         unsafe {
-            let foot = self.footer?;
-            let foot = foot.as_ref();
+            let foot = self.footer?.as_ref();
```

Comment:
This is the only code change, I hope this is ok, since this is a documentation PR.